litex_boards/platforms/digilent_zybo_z7.py

```diff
@@ -102,6 +172,22 @@
         "PCW_USB0_RESET_ENABLE"         : "1",
         "PCW_USB0_RESET_IO"             : "MIO 46",
         "PCW_EN_USB0"                   : "1"
+    },
+    "original" : {
+        "PCW_UIPARAM_DDR_PARTNO"        : "MT41K128M16 RE-125",
+        "PCW_FPGA_FCLK0_ENABLE"         : "1",
+        "PCW_UART1_BAUD_RATE"           : "115200",
+        "PCW_EN_UART1"                  : "1",
+        "PCW_UART1_PERIPHERAL_ENABLE"   : "1",
+        "PCW_UART1_UART1_IO"            : "MIO 48 .. 49",
+        "PCW_PRESET_BANK1_VOLTAGE"      : "LVCMOS 1.8V",
+        "PCW_USE_M_AXI_GP0"             : "1",
+        "PCW_USE_S_AXI_GP0"             : "1",
+        "PCW_USB0_PERIPHERAL_ENABLE"    : "1",
+        "PCW_USB0_USB0_IO"              : "MIO 28 .. 39",
+        "PCW_USB0_RESET_ENABLE"         : "1",
+        "PCW_USB0_RESET_IO"             : "MIO 46",
+        "PCW_EN_USB0"                   : "1"
```

Comment:
Both have same content: maybe simplify using existing one to avoid code duplication?